index.html

```diff
@@ -0,0 +1,109 @@
+    <link rel="manifest" href="dist/images/site.webmanifest">
+    <link rel="mask-icon" href="dist/images/safari-pinned-tab.svg" color="#5bbad5">
+    <meta name="msapplication-TileColor" content="#da532c">
+    <meta name="theme-color" content="#ffffff">
+    <link href="https://fonts.googleapis.com/css?family=Heebo:400,700|IBM+Plex+Sans:600" rel="stylesheet">
+    <link rel="stylesheet" href="dist/css/style.css">
+    <script src="https://unpkg.com/scrollreveal@4.0.0/dist/scrollreveal.min.js"></script>
+</head>
+
+<body class="is-boxed has-animations lights-off">
+    <div class="body-wrap boxed-container">
+        <header class="site-header">
+            <div class="container">
+                <div class="site-header-inner">
+                    <div class="brand header-brand">
+                        <h1 class="m-0">
+                            <div class="lights-toggle">
+                                <input id="lights-toggle" type="checkbox" name="lights-toggle" class="switch">
+                                <label for="lights-toggle" class="text-xs"><span>Turn me <span
+                                            class="label-text">dark</span></span></label>
+                            </div>
+                        </h1>
+                    </div>
+                </div>
+            </div>
+        </header>
+
+        <main>
+
+            <section class="features section">
+                <div class="container">
+                    <div class="features-inner section-inner has-bottom-divider">
+                        <div class="features-wrap">
+                            <div class="feature is-revealing">
+                                <div class="feature-inner">
+                                    <a href="//docs.sublimetext.io" class="feature-icon">
+                                        <img class="asset-light" src="dist/images/sublime_text_1024x1024x32.png"
+                                            alt="Sublime Text Documentation">
+                                        <img class="asset-dark" src="dist/images/sublime_text_1024x1024x32.png"
+                                            alt="Sublime Text Documentation">
+                                    </a>
+                                    <div class="feature-content">
+                                        <a href="//docs.sublimetext.io">
+                                            <h3 class="feature-title mt-0">Unofficial Sublime Text Documentation</h3>
+                                        </a>
+                                        <p class="text-sm mb-0">The Sublime Text Community Documentation is a community
+                                            effort at documenting the <a href="https://sublimetext.com/">Sublime
+                                                Text</a> text editor and accompanying its <a
+                                                href="https://sublimetext.com/docs/3/">official documentation</a>.</p>
```

Comment:
These are some weird line breaks to be honest, but it's not like it will be changed much or similar.